simulation/sub8_gazebo/worlds/a_whole_new.world

```diff
@@ -17,7 +17,7 @@
       <uri>model://sun</uri>
     </include>
 
-    <!-- Never delete this -->
+    <!-- Never delete this, uncomment to bring back the ghost cessna
```

Comment:
Ahhh I didn't catch your loophole last PR. It says don't delete, but it never said anything about commenting out :)